apps/api/src/server/utils/published-fields-changed.js

```diff
@@ -0,0 +1,53 @@
+import * as caseRepository from '../repositories/case.repository.js';
+import BackOfficeAppError from './app-error.js';
+import { buildNsipProjectPayload } from '../applications/application/application.js';
+
+/** @typedef {import('@pins/applications.api').Schema.Case} Case */
+
+/**
+ * Checks whether the given column name is included when the case is published
+ *
+ * @param {Case} original
+ * @param {Case} updated
+ * @returns {boolean}
+ * */
+const publishedFieldsHaveChanged = (original, updated) => {
```

Comment:
I think its worth making it clear in the name this is for cases, not generic.

`publishedCaseFieldsHaveChanged` perhaps?

Reply:
That's done